wordpress-mobile/WordPress-Android
# Stats - Insights architecture

Fixes #8324 

Creates a presentation layer to show Insights in the refreshed Stats. The list of insights is coming from FluxC. This implementation shows a `NotImplemented` block as a placeholder of all the mandatory insight items.

All the items loading should be implemented in `InsightsDomain`. All the calls will be triggered in parallel.

As another PR we'll add more List functionality into `InsightsViewModel` and `InsightsFragment`, for example pull to refresh, empty state and loading state.  

I'm not adding a milestone since the target of this PR is a readonly branch.

To test:
- Open Stats from the main menu
- See a list of placeholder items
